README.md

```diff
@@ -1,3 +1,96 @@
-## My test task
+# OrderBook
+
+---
 
 Here you can find my implementation of OrderBook.
+
+## Background:
+Imagine that you are in a market which only trades one type of item (e.g. tomatoes of equal quality;
+generally known as "shares"). 
+There will be a certain amount of tomatoes (shares) being offered at certain prices. 
+Also, there will be people willing to buy at a certain price. 
+Imagine that everyone who could buy/sell at an acceptable price(limit price) immediately does that and 
+leaves the market. 
+This way (most of the time), nobody can perform 
+a trade right now and everyone has to wait until something changes (e.g. someone reconsiders "acceptable" 
+price, or a new person appears). 
+Those "limit orders" (people willing to buy/sell in certain quantities) are the limit order book. 
+In some cases people are willing to buy/sell at any price (that's called a "market order"), such a person 
+is always going to perform the trade and then leaves the market.
+In other words, each price level (for simplicity let's think of it as an integer value) can be either 
+bid (there are people willing to buy at this price), 
+ask (people are willing to sell at this price) or spread (nobody is willing to buy or sell at this price).
+Generally, the order book looks like the following example (B - bid, S - spread, A - ask), 
+size defines how many shares can be bought/sold at this price:
+
+## Build:
+To build and compile the project, follow these steps:
+- You need linux OS;
```

Comment:
You have the same line in the `Pre-requisites` section

Reply:
fix